findip.py

```diff
@@ -112,7 +121,7 @@ def getContentIP(targetURL, localIP, networkDeviceID):
         if ipMap[ip] > maxIPCount:
             maxIPCount = ipMap[ip]
             maxIP = ip
-    if maxIPCount < len(dumpOutput) / 2:
+    if maxIPCount < (len(dumpOutput) - nonMatches) / 2:
```

Comment:
On tcpdump the output seemed to get split (the first bit with the IP and then the trailing bit on a new line) so this check would fail. This seemed like the easiest fix.